tasks/interactive-bots.md

```diff
@@ -168,15 +180,28 @@ messages to users.
                                          'from %s:' % (original_sender,))
   ```
 
+
+* Edit the `client.send_message` in `handle_message`:
+
+  ```
+  client.send_message(dict(
+      type='stream',
+      to='links',
+      subject=message['sender_email'],
+      content=new_content,
+  ))
+  ```
+
+
 * Create a links bot in settings for an existing user. Use its credentials in
 `~/.zuliprc-local`. Subscribe the bot to `devel` and `social` streams.
 
 * Check that the bot is working as expected:
 
   - it reacts to the `@link` and `@resource` at the beginning of a message
 
-  - it currently posts the message to the followup stream, as the followup bot
-  would
+  - it currently posts the message to the links stream, as the followup bot
+  would post the message to the followup stream
```

Comment:
As pointed in the previous comment - this would change the intention of the task description.